uniswap_example.py

```diff
@@ -18,17 +18,39 @@
         "--n_steps", type=int, default=100, help="Number of steps of the simulation"
     )
     parser.add_argument("--sigma", type=float, default=0.3, help="GBM volatility")
+    parser.add_argument("--mu", type=float, default=0.0, help="GBM drift")
     parser.add_argument(
         "--batch_runner",
         action="store_true",
         help="Run batch of simulations over different simulation parameters",
     )
+    parser.add_argument(
+        "--cache",
+        action="store_true",
+        help="Generate a new request cache file.",
+    )
+    parser.add_argument(
+        "--alchemy_key",
+        type=str,
+        help="Generate a new request cache file.",
+    )
     args = parser.parse_args()
 
     with open(os.path.join("verbs_examples", "uniswap", "cache.json"), "r") as f:
         cache_json = json.load(f)
 
-    cache = verbs.utils.cache_from_json(cache_json)
+    if args.cache:
+        assert (
+            args.alchemy_key is not None
+        ), "Alchemy key required, set with '--alchemy_key' argument"
+        cache = sim.init_cache(
+            args.alchemy_key,
+            19163600,
```

Comment:
Do we want to make the block number configurable?

Reply:
added in a new commit